### PR TITLE
Initial swagger docs

### PR DESCRIPTION
**Pull request by cfeeney-va**
Will most likely change, but sets up swagger and shows how documentation will work.

### CODE REVIEW

**Comment by saneshark** [comment]
Oh cool @cfeeney-va . It looks like this gem allows you to put the documentation right in line with the controller. I wonder if it has any mechanism or reading specs too? I'm going to be writing some specs soon, I can start writing it in whatever format it uses to determine JSON output if you guys want.

File: app/controllers/rx/v1/prescriptions_controller.rb [hunk 1]
```diff
@@ -4,10 +4,22 @@ class PrescriptionsController < ApplicationController
       respond_to :json # add xml, etc?
       before_action :authenticate_client, only: :prescription
 
+      swagger_controller :prescriptions, "Prescriptions"
+
+      swagger_api :history do
+        summary "Returns all VA prescriptions for the patient that were fillable online"
```
Comment (ayaleloehr) [inline app/controllers/rx/v1/prescriptions_controller.rb]: change "were" to "are" to keep it present tense?

Comment (ayaleloehr) [inline app/controllers/rx/v1/prescriptions_controller.rb]: Maybe change "were" to "are currently or were at one point" or something to make explicit that history is both Active and past?  Feel free to change that verbiage.

File: app/controllers/rx/v1/prescriptions_controller.rb [hunk 2]
```diff
@@ -4,10 +4,22 @@ class PrescriptionsController < ApplicationController
...
+        notes "This is the patient's entire prescription history, including active prescriptions"
+      end
+
       def history
         mhv_api_call("history", params[:id])
       end
 
+      swagger_api :prescription do
+        summary "Returns all current VA prescriptions for the patient that are fillable online"
+        notes "This only includes the patient's active prescriptions. Pass id to retrieve one rx."
```
Comment (ayaleloehr) [inline app/controllers/rx/v1/prescriptions_controller.rb]: below "ID" is capitalized but here is isn't. either works, but let's pick one and standardize.

**Comment by ayaleloehr** [comment]
@cfeeney-va can you add documentation on how to generate the API docs?  Only a couple other nits and mostly looks good, but want I to try it locally, so please assign back to me when you have added the documentation and I'll test then merge. Thanks!

File: app/controllers/rx/v1/prescriptions_controller.rb (hunk 2 above)
Comment (cfeeney-va) [inline app/controllers/rx/v1/prescriptions_controller.rb]: good catch; updated here

**Comment by cfeeney-va** [comment]
thanks @saneshark; i took a quick look and i think this gem might be useful: https://github.com/domaindrivendev/swagger_rails.  Is that the kind of thing you were thinking of?  I'd love to help out with documentation!

**Comment by cfeeney-va** [comment]
@ayaleloehr to generate docs run: rake swagger:docs
then start your rails server and go to http://localhost:3000/api to view

**Comment by saneshark** [comment]
@cfeeney-va I will try to write prescription-api specs in the manner described there, so that if we do integrate with swagger_rails it's seamless. I figured there was a gem out there that does that, because other gems for different api documentation facilities have something similar and it is super helpful. Thanks for that link!

**Comment by ayaleloehr** [comment]
Thanks for the updates!  LGTM. I added in a couple lines in the README, can you make sure you agree with them and then merge this PR @cfeeney-va ?

**Comment by cfeeney-va** [comment]
LGTM

**Pull request merged**